security/tpm/src/Microsoft.Azure.Devices.Provisioning.Security.Tpm.csproj

```diff
@@ -26,7 +26,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <VersionPrefix>1.0.1</VersionPrefix>
+    <VersionPrefix>1.0.2</VersionPrefix>
```

Comment:
Same questions as above - why do we need to release Transports and Security? Was there a fix?

Reply:
same as above.